script/main.js

```diff
@@ -0,0 +1,59 @@
+const shelf = document.getElementById('shelf');
+const bookItem = document.getElementById('bookItem');
+
+const item = document.createElement('div');
+item.classList.add('book');
+
+const form = document.getElementById('form');
+const inputTitle = document.getElementById('title');
+const inputAuthor = document.getElementById('author');
+
+const store = localStorage.getItem('store') ? JSON.parse(localStorage.getItem('store')) : [];
+
+function removeBook(book) {
+  const bookItems = Array.from(document.getElementsByClassName('book'));
+  Object.keys(bookItems).forEach((key) => {
+    if (book === key) {
+      bookItems[key].remove();
+      store.splice(key, 1);
+      localStorage.setItem('store', JSON.stringify(store));
+    }
+  });
+}
+
+function display() {
+  shelf.innerHTML = '';
+  store.forEach((currentBook) => {
+    item.innerHTML = `<p id="book-title">${currentBook.title}</p>
+    <p id="book-aurthor">${currentBook.author}</p>
+    <button type="button" class="remove-btn">Remove</button>
+    <hr/>`;
+    bookItem.appendChild(item.cloneNode(true));
+    shelf.appendChild(bookItem);
+  });
+
+  const removeButton = Array.from(document.getElementsByClassName('remove-btn'));
+  Object.keys(removeButton).forEach((removeKey) => {
+    removeButton[removeKey].addEventListener('click', () => {
+      removeBook(removeKey);
+    });
+  }, false);
+}
+
+display();
+
+function addBook(e) {
```

Comment:
- [ ] Good job so far! But your function that adds a new book to the collection is not properly working. It is adding two books at the same time when you try for the second time and more. Let's see the below gif 👇 
![20220110_164619_DURecorder_AdobeCreativeCloudExpress](https://user-images.githubusercontent.com/67087664/148763426-c168e258-ab39-4fef-bfab-8806f1bbad2d.gif)

Please, kindly fix it! 🙏